test/problematic_tests/impl_test.py

```diff
@@ -0,0 +1,58 @@
+import os
+import pytest
+import traceback
+import unittest
+
+from configparser import ConfigParser
+
+from GenomeFileUtil.GenomeFileUtilImpl import GenomeFileUtil
+
+
+def assert_exception_correct(got: Exception, expected: Exception):
```

Comment:
This is probably going to be needed in other test functions and at that point should be moved to a shared file

Reply:
Updated!

Reply:
:+1: